packages/styles/src/menu-dropdown/menu-dropdown.scss

```diff
@@ -0,0 +1,95 @@
+@import "../core/_dev";
+
+.as-menu-dropdown {
+  --as-dropdown-menu-hover-color: #D3F6E6;
```

Comment:
Shouldn't we put a SASS variable to set the colour in the CSS variable?

Reply:
since this color is only used here I'm not sure if we should create a sass variable